### PR TITLE
fix navbar

### PR DESCRIPTION
**Pull request by mistahuman**
vez use this commit to fix your buggy navbar! (and requirements python)

### CODE REVIEW

File: be/requirements.txt
Comment (GiacomoCortesi) [inline be/requirements.txt:1]: I'd avoid using pip freeze to write requirements file as it also adds dependencies not directly used in the project.
I'd rather maintain it manually for greater control, or use something as pipreqs (https://github.com/bndr/pipreqs) to write to requirements only the packages that are actually in use, avoiding the risk to pollute the requirements file with unwanted packages.

Comment (rollomista) [reply be/requirements.txt:1]: Awesome!

File: ui/components/navbar.tsx
Comment (GiacomoCortesi) [inline ui/components/navbar.tsx:1]: well done bro

**Review by GiacomoCortesi — APPROVED** [review]
well done bro

**Pull request merged**